### PR TITLE
297 roles

### PR DESCRIPTION
**Pull request by gentee**
Checked role tables

### CODE REVIEW

**Review by ghost — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Review by ghost — APPROVED** [review]

File: packages/migration/ecosysystem.go [hunk 1]
```diff
@@ -904,58 +925,64 @@ var (
...
-			"delete"    boolean,
+			"delete"    bigint NOT NULL DEFAULT '0',
 			"role_type" bigint NOT NULL DEFAULT '0',
 			"creator_id" bigint NOT NULL DEFAULT '0',
 			"date_create" timestamp,
 			"date_delete" timestamp,
 			"creator_name"	varchar(255) NOT NULL DEFAULT '',
-			"creator_avatar" varchar(255) NOT NULL DEFAULT ''
+			"creator_avatar" bytea NOT NULL DEFAULT '',
+			"rb_id" bigint NOT NULL DEFAULT '0'
```
Comment (potehinre) [inline packages/migration/ecosysystem.go]: rb_id is obsolete

File: packages/migration/ecosysystem.go [hunk 2]
```diff
@@ -877,24 +898,24 @@ var (
...
 			"finished_processing_id" bigint NOT NULL DEFAULT '0',
 			"finished_processing_time" timestamp,
-			"page_name"	varchar(255) NOT NULL DEFAULT ''
+			"role_id"	bigint NOT NULL DEFAULT '0',
+			"role_name"	varchar(255) NOT NULL DEFAULT '',
+			"recipient_id"	bigint NOT NULL DEFAULT '0',
+			"started_processing_id"	bigint NOT NULL DEFAULT '0',
+			"body_text"	text NOT NULL DEFAULT '',
+			"header_text"	text NOT NULL DEFAULT '',
+			"rb_id" bigint NOT NULL DEFAULT '0'
```
Comment (potehinre) [inline packages/migration/ecosysystem.go]: rb_id is obsolete

File: packages/migration/ecosysystem.go [hunk 3]
```diff
@@ -904,58 +925,64 @@ var (
...
-			"member_avatar"	varchar(255) NOT NULL DEFAULT '',
+			"member_name" varchar(255) NOT NULL DEFAULT '',
+			"member_avatar"	bytea NOT NULL DEFAULT '',
 			"appointed_by_id" bigint NOT NULL DEFAULT '0',
 			"appointed_by_name"	varchar(255) NOT NULL DEFAULT '',
 			"date_start" timestamp,
 			"date_end" timestamp,
-			"delete" boolean
+			"delete" bigint NOT NULL DEFAULT '0',
+			"rb_id" bigint NOT NULL DEFAULT '0'
```
Comment (potehinre) [inline packages/migration/ecosysystem.go]: rb_id is obsolete

**Review by potehinre — CHANGES_REQUESTED** [review]

File: packages/migration/ecosysystem.go (hunk 2 above)
Comment (gentee) [reply packages/migration/ecosysystem.go]: Removed all rb_id

**Review by potehinre — APPROVED** [review]

**Pull request merged**